docs/known_issues.rst

```diff
@@ -75,6 +75,39 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Quantities Lose Their Units with Operations on pandas.Series
+------------------------------------------------------------
```

Comment:
This might be true, but there are no examples about that. A title that would actually match the text would be
```suggestion
Multiplying a `pandas.Series` with an `~astropy.units.Unit` does not produce a |Quantity|
-----------------------------------------------------------------------------------------
```